UI/GameInfoCache.cpp

```diff
@@ -365,7 +365,7 @@ class GameInfoWorkItem : public Task {
 		}
 
 		// In case of a remote file, check if it actually exists before locking.
-		if (!info_->GetFileLoader()->Exists()) {
+		if (!info_->GetFileLoader() || !info_->GetFileLoader()->Exists()) {
```

Comment:
Hm.  I think we now schedule these on multiple threads.  There might be a race condition now where `DisposeFileLoader()` could be run by one thread while another is starting up for the very same file (i.e. with additional want flags.)

-[Unknown]

Reply:
Ah, that's a good catch, very likely what happens..